packages/terra-clinical-item-view/tests/wdio/item-view-spec.js

```diff
@@ -0,0 +1,46 @@
+/* global before, browser, Terra */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+const selector = '#root';
+
+describe('Item View', () => {
+  describe('Displays a clinical item view with default props', () => {
+    before(() => {
+      browser.url('/#/raw/tests/clinical-item-view/default-item-view');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Displays a clinical item view with displays present', () => {
```

Comment:
Can we make these describes a bit less verbose? Something like `Clinical Item View with Displays`?